enhancements/update/cluster-profiles.md

```diff
@@ -105,6 +92,149 @@ the required manifests while keeping a single master source.
 The current installation profile is called `self-managed-high-availability`. All current
 manifests must specify it. Future profiles may choose.
 
+#### Design 1
+
+A cluster profile is specified to the CVO as an identifier an environment variable.
+
+```
+CLUSTER_PROFILE=[identifier]
+```
+
+For a given cluster, only one cluster profile may be in effect.
+
+The profile is a set-once property and cannot be changed once the cluster has started.
+
+The cluster-version-operator picks the value for the cluster profile according to this order:
+* if it is defined and not empty, the environment variable,
+* otherwise, the default profile `self-managed-high-availability`.
+
+Clusters in a version unaware of the cluster profile must upgrade to the `self-managed-high-availability` profile.
+
+When upgrading, outgoing CVO will forward the cluster profile information to the incoming CVO with the environment variable.
+
+`include.release.openshift.io/[identifier]=true` would make the CVO render this manifest only when `CLUSTER_PROFILE=[identifier]`
+has been specified. 
+
+##### Usage
```

Comment:
i think this usage section is not accurate.

right now, the managed ibm cloud offering renders the cvo itself and is not a client of openshift-install.

Reply:
yes sorry, this is what I meant. Bad wording/organization. It should be more precise here.

Reply:
It should be better now.